cli/tsc.rs

```diff
@@ -44,32 +44,38 @@ pub fn compiler_snapshot() -> Snapshot {
   Snapshot::Static(COMPILER_SNAPSHOT)
 }
 
-/// Provide static assets that are not preloaded in the compiler snapshot.
+macro_rules! inc {
+  ($e:expr) => {
+    include_str!(concat!("dts/", $e))
+  };
+}
+
+lazy_static! {
+  /// Contains static assets that are not preloaded in the compiler snapshot.
+  pub(crate) static ref STATIC_ASSETS: HashMap<&'static str, &'static str> = [
+    ("lib.dom.asynciterable.d.ts", inc!("lib.dom.asynciterable.d.ts")),
+    ("lib.dom.d.ts", inc!("lib.dom.d.ts")),
+    ("lib.dom.iterable.d.ts", inc!("lib.dom.iterable.d.ts")),
+    ("lib.es6.d.ts", inc!("lib.es6.d.ts")),
+    ("lib.es2016.full.d.ts", inc!("lib.es2016.full.d.ts")),
+    ("lib.es2017.full.d.ts", inc!("lib.es2017.full.d.ts")),
+    ("lib.es2018.full.d.ts", inc!("lib.es2018.full.d.ts")),
+    ("lib.es2019.full.d.ts", inc!("lib.es2019.full.d.ts")),
+    ("lib.es2020.full.d.ts", inc!("lib.es2020.full.d.ts")),
+    ("lib.esnext.full.d.ts", inc!("lib.esnext.full.d.ts")),
+    ("lib.scripthost.d.ts", inc!("lib.scripthost.d.ts")),
+    ("lib.webworker.d.ts", inc!("lib.webworker.d.ts")),
+    ("lib.webworker.importscripts.d.ts", inc!("lib.webworker.importscripts.d.ts")),
+    ("lib.webworker.iterable.d.ts", inc!("lib.webworker.iterable.d.ts")),
+  ]
+  .iter()
```

Comment:
Can you use `into_iter` here to avoid the `.copied()`?

Reply:
That doesn't work, because `.into_iter()` returns a `FromIterator<&(&str, &str)>` which `HashMap<&str, &str>` can't be built from.  The only way to build a static `HashMap` like this is to do `.copied()`.

Reply:
👍 - then feel free to merge

Reply:
I think you should be able to do `&[ /* .. */ ].into_iter().cloned().collect()` here?

Reply:
`(&[ /* .. */ ]).into_iter().cloned().collect()` works, if remediated the other two static hashmaps we've got.